app/controllers/coronavirus_form/accessibility_statement_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::AccessibilityStatementController < ApplicationController
+  def show
```

Comment:
Bill add a [change](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/app/controllers/application_controller.rb#L19-L21) yesterday which means you no longer need to add a show method to every controller.

You will need to add a [`skip_before_action`](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/app/controllers/coronavirus_form/privacy_controller.rb#L4) so that users can view this page without having answered any questions.

Reply:
Yeah, this was raised a day before that. Will rebase and update, thank you!